buildenv/Build_Instructions_V8.md

```diff
@@ -0,0 +1,322 @@
+
+```
+cd /<my_home_dir>
+wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz \
+tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2 \
+rm -f freemarker.tgz
+```
+
+### 2. Get the source
+:blue_book:
+First you need to clone the Extensions for OpenJDK for OpenJ9 project. This repository is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches. Run the following command:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk8
+```
+Cloning this repository can take a while because OpenJDK is a large project! When the process is complete, change directory into the cloned repository:
+```
+cd openj9-openjdk-jdk8
+```
+Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+
+```
+bash ./get_source.sh
+```
+### 3. Configure
+:blue_book:
+When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
+```
+bash ./configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar /
+                 --with-cups-include=<cups_include_path> /
+                 --disable-warnings-as-errors
+```
+where `<my_home_dir>` is the location where you stored **freemarker.jar** and `<cups_include_path>` is the absolute path to CUPS. For example `/opt/freeware/include`.
+
+### 4. build
+:blue_book:
+Now you're ready to build OpenJDK with OpenJ9:
+```
+make all
+```
+:warning: If you just type `make`, rather than `make all` your build will fail, because the default `make` target is `exploded-image`. If you want to specify `make` instead of `make all`, you must add `--default-make-target=images` when you run the configure script. For more information, read this [issue](https://github.com/ibmruntimes/openj9-openjdk-jdk9/issues/34).
+
+Two Java builds are produced: a full developer kit (jdk) and a runtime environment (jre)
+- **build/aix-ppc64-normal-server-release/images/jdk**
+- **build/aix-ppc64-normal-server-release/images/jre**
+
+    :pencil: A JRE binary is not currently generated due to an OpenJDK bug.
+
+
+### 5. Test
+:blue_book:
+For a simple test, try running the `java -version` command.
+Change to the /jdk directory:
+```
+cd build/aix-ppc64-normal-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "1.8.0-internal"
+OpenJDK Runtime Environment (build 1.8.0-internal-admin_2017_10_31_10_46-b00)
+Eclipse OpenJ9 VM (build 2.9, JRE 8 AIX ppc-64 Compressed References 20171031_000000 (JIT enabled, AOT enabled)
+OpenJ9   - 68d6fdb
+OMR      - 7c3d3d7
+OpenJDK  - 27f5b8f based on jdk8u152-b03)
+```
+:blue_book: *Congratulations!* :tada:
+
+----------------------------------
+
+## Windows
+:ledger:
+
+:construction:
+We haven't created a full build process for Windows yet? Watch this space!
```

Comment:
Not urgent, but these would be great places to link to the issues opened to produce Windows, Macos, ARM, etc. builds...

Windows: #55
Macos: #36 
ARM: #53 #66 #138

Reply:
The v9 instructions could be updated similarly. I'll do both as an enhancement on a separate PR if that's ok?